test/connectors/ApprovalConnectorSpec.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.BaseSpec
+import mocks.{MockAppConfig, MockHttpClient}
+import models.errors.InternalServerError
+import models.{RequestOutcome, ApprovalResponse}
+import play.api.libs.json.{JsValue, Json}
+import play.api.test.{DefaultAwaitTimeout, FutureAwaits}
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class ApprovalConnectorSpec extends BaseSpec {
+
+  private trait SubmitForApprovalTest extends MockHttpClient with FutureAwaits with DefaultAwaitTimeout {
+
+    implicit val hc: HeaderCarrier = HeaderCarrier()
+
+    val connector: ApprovalConnector = new ApprovalConnector(mockHttpClient, MockAppConfig)
+    val endpoint: String = MockAppConfig.externalGuidanceBaseUrl + "/external-guidance/submitted"
+
+    val id: String = "Oct90005"
+    val dummyProcess: JsValue = Json.obj("processId" -> id)
+
+  }
+
+  "Calling method makeAvailableForApproval with a dummy process" should {
+
+    "Return an instance of the class SaveSubmittedProcessResponse for a successful call" in new SubmitForApprovalTest {
```

Comment:
As with ScratchConnectorSpec the class name in the test description is out of date.

Reply:
Updated in latest commit